azext_iot/iothub/commands_state.py

```diff
@@ -15,10 +15,10 @@ def state_export(
     resource_group_name: Optional[str] = None,
     login: Optional[str] = None,
     auth_type_dataplane: Optional[str] = None,
-    overwrite_file: Optional[bool] = False
+    force: bool = False
 ):
     sp = StateProvider(cmd=cmd, hub=hub_name, rg=resource_group_name, login=login, auth_type_dataplane=auth_type_dataplane)
-    sp.save_state(filename, overwrite_file)
+    sp.save_state(filename, force)
```

Comment:
This can be done in a future iteration but I see two pivots from here in overall design:
- A user should be able to choose to not include aspects of their hub for export.
  - i.e. I have endpoints and devices but I only want to export (or migrate) devices.
- We should return a dictionary corresponding to the export data structure to make use of standard Az CLI features (like output modifiers i.e. yaml format). Also its standard shell practice to redirect output from stdout to a file. As in the filename/--state-file argument is not a necessity but a nice to have.

Reply:
I'll put it in the presentation